src/routes/trips/[tripId]/players/+page.server.ts

```diff
@@ -76,15 +75,22 @@ export const actions: Actions = {
 		try {
 			const { tripId, playerId } = deleteSchema.parse(data);
 
-			const { error: pgError } = await supabaseClient
+			const { error: tripPlayerError } = await supabaseClient
 				.from('trip_players')
 				.delete()
 				.eq('trip_id', tripId)
 				.eq('player_id', playerId);
 
-			if (pgError) return error(500, pgError.message);
+			if (tripPlayerError) return fail(500, {message: tripPlayerError.message});
+
+			const { error: playerError } = await supabaseClient
+				.from('players')
+				.delete()
+				.eq('id', playerId);
```

Comment:
In the future, removing a player from a trip should only remove the `trip_players` entry. I'm removing the player from the `players` table for now too so we don't get into a state with tons of duplicate players until we support "adding an existing bunkie bois player to this trip"